.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
Is this file necessary to be checked in?

Reply:
I'm not too sure actually. But I thought it'd be nice to check this in so anyone can deploy to AppEngine without making their own version of this file.